### PR TITLE
add pandarallel

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge/staged-recipes please review

File: recipes/pandarallel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pandarallel-{{ version }}.tar.gz
+  sha256: 3fa7d9baf79ad49e71db923a924a127f2b06a2b45d7229b47655fbcb620929b2
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (chrisburr) [inline recipes/pandarallel/meta.yaml:15]: This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

Comment (hadim) [reply recipes/pandarallel/meta.yaml:15]: The lib does not work on Windows system. It means people will be able to install it but it won't work. Is that still ok?

Comment (chrisburr) [reply recipes/pandarallel/meta.yaml:15]: I think that’s fine I’m this case

**Comment by hadim** [comment]
@chrisburr LGTM

File: recipes/pandarallel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pandarallel-{{ version }}.tar.gz
+  sha256: 3fa7d9baf79ad49e71db923a924a127f2b06a2b45d7229b47655fbcb620929b2
+
+build:
+  number: 0
+  noarch: true
```
Comment (chrisburr) [inline recipes/pandarallel/meta.yaml:15]:
```suggestion
  noarch: python
```

Comment (hadim) [reply recipes/pandarallel/meta.yaml]: oups sorry xD

**Pull request merged**